app/helpers/file_set_helper.rb

```diff
@@ -15,12 +15,20 @@ def render_collection_list(fs)
   end
 
   def display_multiple(value)
-    auto_link(value.join(" | "))
+    # TODO: figure out why sometimes we get an string and some others an array
+    if value.respond_to?("join")
+      string_value = value.join(" | ")
+    else
+      string_value = value
+    end
+    auto_link(string_value)
   end
 
   private
 
     def render_visibility_badge
+      # TODO: figure out why #file_set is sometimes nil
+      return unless @file_set
```

Comment:
I was able to remove the TODO and the guard clause for this once I figure out what this code was doing.